src/components/ui/Input.tsx

```diff
@@ -82,7 +83,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(function (
   );
 
   return (
-    <div data-testid="input" className="relative">
+    <div data-testid={testId} className="relative">
       <div className={inputComponentClassName}>
         {label && <label className={labelClasssName}>{label}</label>}
         <input
```

Comment:
wouldn't it be great to add the test-id on the `<input/>`

Reply:
I think from the parent it's much easier to have access to all the children inside, in that case we don't need an id for each child we need to test.